libs/scheduler/aws.go

```diff
@@ -45,6 +45,10 @@ func (job *Job) PopulateAwsCredentialsEnvVarsForJob() error {
 		log.Printf("Project-level AWS role detected, Assuming role for project: %v", job.ProjectName)
 		var err error
 		backendConfigArgs, err := populateretrieveBackendConfigArgs(*job.StateEnvProvider)
+
+		// Terragrunt will cause a backend configuration problem if backend-config options are passed and envs of the same key are passed. 
+		// which will trigger a request to init with --reconfigure, so do not use backend-config for terragrunt	
+		if job.Terragrunt != true {
```

Comment:
nitpick: the body of this if statement is not indented properly

Reply:
This is me not having automatic go formatting on this project.  I use it on others so I get lazy as normally when I save the file it just fixes it all.  I will take care of it.